GVRf/Framework/framework/src/main/jni/engine/renderer/gl_renderer.cpp

```diff
@@ -306,6 +311,11 @@ namespace gvr
             GL(glEnable(GL_DEPTH_TEST));
         }
 
+        if (!render_data->depth_mask())
+        {
+            GL(glDepthMask(GL_FALSE));
```

Comment:
Assuming the default state is that the depth mask is enabled, shouldn't we enable it from restoreRenderStates?